LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2020-2021, NumPy Developers
```

Comment:
Let's update this to 2010, that's the oldest commit in this repo (just not in the `master` branch). The new site started in 2019.

Reply:
Done.